### DISCUSSION TITLE
Running `osmium` CLI job with `pyosmium`

### DISCUSSION THREAD

**Discussion by manmartgarc** [osmcode/pyosmium#224] [Q&A] [answered]
Hello!

I am wondering if someone could give me some directions how to replicate this `osmium` CLI call within the `pyosmium` package:

``` bash
osmium export -o data/osm/geojson/SYC-latest-linestring.geojson -c config.json -O data/osm/pbf/SYC.osm.pbf --geometry-types=linestring
```

I found a way to filter the tags, but not sure how to also pass the geometry types and multiple tags at the same time.

Thanks in advance for any feedback!

**Comment by lonvia** [comment]
If you want to work with geometries, then you should use shapely. The documentation has a [short section how to get the geometry inside the handler](https://docs.osmcode.org/pyosmium/latest/intro.html#interfacing-with-shapely).

Once you have a Shapely geometry, you can convert it to geojson, add tags as properties and write everything out again.

**Comment by lonvia** [comment, marked as answer]
Just for the fun of it, I've added an example for printing geojson from geometries: https://github.com/osmcode/pyosmium/blob/master/examples/convert_to_geojson.py Turns out you don't even need shapely. If you are only interested in LineStrings, then you only need the `way()` callback.

**Reply by manmartgarc** [reply]
Wow thank you so much! I really appreciate it :)
